eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -670,7 +671,7 @@ object Router {
   case object GetChannelsMap
   case object GetChannelUpdates
 
-  case class GetNode(nodeId: PublicKey)
+  case class GetNode(replyTo: ActorRef, nodeId: PublicKey)
```

Comment:
Can't we make that a `replyTo[GetNodeResponse]`? At the caller's site it will need a conversion that you should have by importing `import akka.actor.typed.scaladsl.adapter._`.

Reply:
Good point; fixed in [bc0e1b9](https://github.com/ACINQ/eclair/pull/2575/commits/bc0e1b9a03081eaba768a9ea61fb8f3a93729ee2)